spec/features/admin/order_shipments_links_to_returns_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.feature 'Return Button Under Order#edit', type: :feature do
+  let!(:order) { create :shipped_order }
+  stub_authorization!
+
+  before :each do
+    visit spree.edit_admin_order_path(order)
+  end
+
+  scenario 'displays the button' do
```

Comment:
Right. But you're mixing DSLs, using `scenario` with `before` and `let`, rather than with `background` and `given`. Like I said, a _very_ fine hair to split.